install/README.md

```diff
@@ -223,6 +223,12 @@ of these settings or provide values to mandatory fields.
   - **Type:** `integer`
   - **Default:** `900`
 
+- **`SS_SILENCED_SYSTEM_CHECKS`**:
+  - **Description:**  comma-separated list of ignored system checks.
+    e.g. mysql.W002,models.W042
```

Comment:
Let's add a link to https://docs.djangoproject.com/en/4.2/ref/settings/#silenced-system-checks here. See examples in `TIME_ZONE` or `SECRET_KEY` above.